cypress/component/Select.cy.jsx

```diff
@@ -0,0 +1,52 @@
+import Select from "@/components/Select";
```

Comment:
add a test that selects an option and then make sure that update actually happened

Reply:
isn't that already in the e2e tests for sort?

Reply:
comp test should make sure that the entire workflow for a comp works. testing the select values and stuff is comp test cause it is handled internally by the comp and not by a parent comp